src/pages/index.js

```diff
@@ -156,23 +157,22 @@ function handleAvatarChange(data) {
       userInfo.setUserInfo(data)
```

Comment:
`avatarPopup.renderLoading(true)` should be in line 154 before the request to change the button text


```js

function handleAvatarChange(data) {
  avatarPopup.renderLoading(true)  // here
  api.updateAvatar(data)
    .then((data) => {
```